data/human/culture conversations.txt

```diff
@@ -163,6 +178,21 @@ mission "Class Trip to Port"
 
 
 
+mission "Old Defense Turrets"
+	minor
+	source
+		attributes "south"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`With an annoyingly long stopover having encouraged you to do a little exploring, you pull the rental car into the designated return parking lot. The unexpected sound of large hydraulic motors draws your attention and you watch as a massive turret lifts up from behind a nearby barricade and turns toward the sky, its single cannon poised to fire at seemingly nothing.`
```

Comment:
Still not onboard with having the player rock up in a car without explanation, especially considering how many spaceports are described as being an area/building close to the landing pads, with New Austria's description even mentioning how the player walks to the centre of the closest town.

Reply:
I'd think that spaceports will at least be heavily connected to the local transportation grid. It wouldn't make sense for you to take the public transport every time you're on a planet.

Reply:
My point isn't that the player wouldn't be able to use/get a rental car, and more that clicking the spaceport button signifies the player going around and searching the spaceport, as opposed to exploring the nearby cities.

Reply:
Wait. Bear with me a moment. Cut doesn't these trigger when you land?

Reply:
No this one requires you to visit the spaceport.

Reply:
Ah...
Hmm.
I'd argue that you can't just show culture around the spaceport alone.

Reply:
Has there been any movement on this issue?

Reply:
Adding a mention that the player first rents a car would fix @Anarchist2 's concern, no? But then the question might be whether the player needs to pay for the car - but I suppose we never see the `<first>` pay for food at a restaurant, so perhaps this can slide too.

Reply:
Given that a copy of the full history of humanity cost a fraction of a credit, I'd say the cost of negligible.

Reply:
Fair. How bout this then?
```suggestion
			`With an annoyingly long stopover having encouraged you to do a little exploring, you rent a car and drive around the immediate area. As you pull the rental car into the designated return parking lot, the unexpected sound of large hydraulic motors draws your attention. You watch as a massive turret lifts up from behind a nearby barricade and turns toward the sky, its single cannon poised to fire at seemingly nothing.`
```

Reply:
I suggest one additional word to Wizrad's.
```suggestion
			`With an annoyingly long stopover having encouraged you to do a little exploring, you rent a car for a short drive around the immediate area. Later, as you pull the car into the designated return parking lot, the unexpected sound of large hydraulic motors draws your attention. You watch as a massive turret lifts up from behind a nearby barricade and turns toward the sky, its single cannon poised to fire at seemingly nothing.`
```

Reply:
Grey's edit works.

Reply:
Agreed!